Travel/Travel/Models/TripParticipant.cs

```diff
@@ -3,8 +3,8 @@
     public class TripParticipant
     {
         public int TripId { get; set; }
-        public Trip Trip { get; set; }
+        public Trip RelatedTrip { get; set; }
```

Comment:
Why are you changing this?

Reply:
Name dispute. It just wouldn't let me migrate, so I had to rename and the issue stopped.

Reply:
Check your models and your relationships at 'OnModelCreating' config. This should not happen